lib/Frontend/FrontendInputs.cpp

```diff
@@ -0,0 +1,175 @@
+//===--- FrontendInputs.cpp -----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/Frontend/FrontendOptions.h"
```

Comment:
The first include in a .cpp file should always be the corresponding header file, both to show which header is "corresponding" and to provide a small check that that header works standalone.

Reply:
Fixed in all my new files. Put a blank line after that include following what seemed to be a convention I saw elsewhere.